easybuild/easyconfigs/p/PCMSolver/PCMSolver-1.1.4-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'PCMSolver'
+version = '1.1.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pcmsolver.readthedocs.org'
+description = """An API for the Polarizable Continuum Model."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'cstd': 'c99'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/PCMSolver/pcmsolver/archive/']
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('zlib', '1.2.8'),
+    ('Boost', '1.61.0', versionsuffix),
+]
+
+builddependencies = [
+    ('CMake', '3.5.2'),
+    ('Eigen', '3.2.7'),
+]
+
+configopts = '-DCMAKE_BUILD_TYPE=Release -DEIGEN3_ROOT=$EBROOTEIGEN -DCMAKE_CXX_FLAGS="$CXXFLAGS $LIBLAPACK_MT -DEIGEN_USE_MKL_ALL"'
```

Comment:
@wpoely86 line too long, please split off the `-DCMAKE_CXX_FLAGS` bit into a new line